plugins/Makefile.am

```diff
@@ -36,7 +36,7 @@ libnoop1_la_SOURCES = H5Znoop1.c H5Zutil.c h5noop.h
 endif #ENABLE_FILTER_TESTING
 
 BUILT_SOURCES = H5Znoop1.c
-CLEANFILES = H5Znoop1.c
+DISTCLEANFILES = H5Znoop1.c
```

Comment:
This should me MOSTLYCLEANFILES.

![image](https://user-images.githubusercontent.com/38856240/83527678-56bce300-a4a5-11ea-9056-d05e86860cff.png)

Reply:
The reasons to use MOSTLY instead of DIST for built sources was explained to us by an autotools power user in netcdf-fortran. The distinction is subtle to me,